index.html

```diff
@@ -1,243 +1,361 @@
-              <li><a href=""><img src="./assets/images/vectors/github.svg" alt=""></a></li>
-              <li><a href=""></a><img src="./assets/images/vectors/M.svg" alt=""></li>
-              <li><a href=""><img src="./assets/images/vectors/chill.svg" alt=""></a></li>
-            </ul>
+            </article>
           </div>
-          <button type="button" class="buttons">Get My Resume</button>
-        </article>
-        <article class="section-b skills">
-          <ul class="skills-list">
-            <li class="skill-item open">
-              <div><span>Languages</span></div>
-              <ul class="skill-set">
-                <li class="skill">
-                  <figure><img src="./assets/images/vectors/js.png" alt=""></figure>
-                  <span>JavaScript</span>
+        </section>
+
+        <section class="about" id="about">
+          <article class="section-a">
+            <h2 class="section-header">About Myself</h2>
+            <p class="description">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <div class="about-socials">
+              <span class="lets-connect">LET’S CONNECT</span>
+              <ul class="socials">
+                <li>
+                  <a
+                    href="https://twitter.com/njorogemwangi21"
+                    target="_blank"
+                    rel="noopener"
+                    ><img src="./assets/images/vectors/twitter.svg" alt=""
+                  /></a>
                 </li>
-                <li class="skill">
-                  <figure><img src="./assets/images/vectors/css.png" alt=""></figure>
-                  <span>CSS</span>
+                <li>
+                  <a
+                    href="https://linkedin.com/in/timothy-njoroge-mwangi"
+                    target="_blank"
+                    rel="noopener"
+                    ><img src="./assets/images/vectors/Linkedin.svg" alt=""
+                  /></a>
                 </li>
-                <li class="skill">
-                  <figure><img src="./assets/images/vectors/html.png" alt=""></figure>
-                  <span>HTML</span>
+                <li>
+                  <a
+                    href="https://github.com/simplegoose"
+                    target="_blank"
+                    rel="noopener"
+                    ><img src="./assets/images/vectors/github.svg" alt=""
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"></a
+                  ><img src="./assets/images/vectors/M.svg" alt="" />
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./assets/images/vectors/chill.svg" alt=""
```

Comment:
- Please ensure all images have alt text, and remember to include the link to the changes you have made in your report 🙏